route/emitter.go

```diff
@@ -56,7 +58,7 @@ func (e *Emitter) Start() {
 func (e *Emitter) emit(batch []*eirini.Routes) {
 	for _, route := range batch {
 		if len(route.Routes) != 0 {
-			err := e.publish(registerSubject, route.Routes, route.Name)
+			err := e.publish(registerSubject, route.Routes, route.Name, route.ServiceAddress, route.ServicePort, route.ServiceTLSPort)
```

Comment:
Instead of passing every field in Route to the `publish` method, we can pass the Route object itself.

cc @mnitchev